flink-ai-flow/lib/airflow/airflow/models/taskinstance.py

```diff
@@ -1195,6 +1195,8 @@ def _run_raw_task(
                 self.set_duration()
                 session.merge(self)
                 return
+            elif self.state == State.KILLED:
+                return
```

Comment:
need raise exception?

Reply:
No need to raise, KILLED state should be treated like SUCCESS and FAILED, just return. But I should move it to 
```
            if self.state in {State.SUCCESS, State.FAILED,}: # add  State.KILLED
                return
```